app/src/main/java/org/glucosio/android/activity/HelloActivity.java

```diff
@@ -100,11 +104,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
         initStartButton();
 
-        Analytics analytics = application.getAnalytics();
-        analytics.reportScreen("Hello Activity");
+        showAnalyticsExplanationDialog();
```

Comment:
We're showing the notice in HelloActivity. This will affect new user only and not current users (since they will jump directly to MainActivity). @elioqoshi how can we handle this from UX prospective? Can we prompt analytics notice and opt-out to existing users right when they open Glucosio after the update?